recipes-devtools/python/python3-jetcam_git.bb

```diff
@@ -0,0 +1,17 @@
+SUMMARY = "Easy to use Python camera interface for NVIDIA Jetson"
+LICENSE = "MIT"
+LIC_FILES_CHKSUM = "file://LICENSE.md;md5=295ba21d3c8eb4396383d735495fbe6e"
+
+inherit setuptools3
+
+SRC_URI = "git://github.com/NVIDIA-AI-IOT/jetcam.git"
+SRCREV = "508ff3a402c5a0449a8d5776c05d2d17d6a87f46"
+PV = "git${SRCPV}"
+
+S = "${WORKDIR}/git"
+
+RDEPENDS_${PN} += "python3-numpy \
```

Comment:
Could you please use `${PYTHON_PN}` instead of `python3` ?

Reply:
Do you think we'd need Python 2 compatibility?  That's where the `${PYTHON_PN}` usage came from.  Looking at the recipes in meta-python, I see some direct use of `python3-` these days.

Reply:
@ichergui Ever since OE-core dropped support for python2, it's OK to use python3 explicitly in recipes in OE-core and meta-openembedded. I'd say let's follow upstream.

Reply:
I can give an example from `meta-openembedded`  concerning this 

http://git.openembedded.org/meta-openembedded/tree/meta-python/recipes-devtools/python/python3-traitlets_5.0.5.bb